wiki/Guides/How_to_Time_Songs/fr.md

```diff
@@ -66,7 +60,7 @@ Maintenant que l'offset est correct, vous pouvez ajuster le BPM. Déterminer l'o
   - Réduisez la vitesse de la musique (75% ou 50% si c'est requis) si vous rencontrez des difficultés.
 - Réécoutez la musique est regardez les battements du métronome.
 
-Pour regarder si le BPM est correct ou non, allez directement vers le milieu ou la fin de la musique, là où la différence entre le tempo de la musique et celui du métronome est accumulée et est facilement audible. Souvent, le BPM va être un nombre entier (pas de valeurs décimales) donc essayez d'ajuster le BPM jusqu'au nombre entier le plus proche pour voir si cela fonctionne dans votre cas. Après cela, vous pouvez essayer de faire des ajustements plus précis comme 101.200 au lieu de 101BPM, voir éventuellement 101.238, pour nommer quelques exemples. Avant de laisser le BPM avec des valeurs dans les centièmes (.01) ou dizaines (.1), enlevez les décimales ou arrondissez les puis réécoutez le métronome. Si la différence est insignifiante ou si vous la trouvez plus précise (regardez le milieu et la fin encore une fois), vous pouvez le garder et considérer le problème comme réglé. Attention : Toutes les musiques électroniques n'ont pas de BPM complètement plat, en dépit d'à quel point elle peuvent sembler banales. Enlever les décimales peut résulter en une erreur de BPM, dans ce cas-là.
+Pour regarder si le BPM est correct ou non, allez directement vers le milieu ou la fin de la musique, là où la différence entre le tempo de la musique et celui du métronome est accumulée est facilement audible. Souvent, le BPM va être un nombre entier (pas de valeurs décimales) donc essayez d'ajuster le BPM jusqu'au nombre entier le plus proche pour voir si cela fonctionne dans votre cas. Après cela, vous pouvez essayer de faire des ajustements plus précis comme 101.200 au lieu de 101BPM, voir éventuellement 101.238, pour nommer quelques exemples. Avant de laisser le BPM avec des valeurs dans les centièmes (.01) ou dizaines (.1), enlevez les décimales ou arrondissez les puis réécoutez le métronome. Si la différence est insignifiante ou si vous la trouvez plus précise (regardez le milieu et la fin encore une fois), vous pouvez le garder et considérer le problème comme réglé. Attention : Toutes les musiques électroniques n'ont pas de BPM complètement plat, en dépit de quel point elle peuvent sembler banales. Enlever les décimales peut résulter en une erreur de BPM, dans ce cas-là.
```

Comment:
not sure if ".1" is specific to english, i've at least never seen it in any other language

```suggestion
Pour regarder si le BPM est correct ou non, allez directement vers le milieu ou la fin de la musique, là où la différence entre le tempo de la musique et celui du métronome est accumulée est facilement audible. Souvent, le BPM va être un nombre entier (pas de valeurs décimales) donc essayez d'ajuster le BPM jusqu'au nombre entier le plus proche pour voir si cela fonctionne dans votre cas. Après cela, vous pouvez essayer de faire des ajustements plus précis comme 101,200 au lieu de 101 BPM, voir éventuellement 101,238, pour nommer quelques exemples. Avant de laisser le BPM avec des valeurs dans les centièmes (0,01) ou dizaines (0,1), enlevez les décimales ou arrondissez les puis réécoutez le métronome. Si la différence est insignifiante ou si vous la trouvez plus précise (regardez le milieu et la fin encore une fois), vous pouvez le garder et considérer le problème comme réglé. Attention : Toutes les musiques électroniques n'ont pas de BPM complètement plat, en dépit de quel point elle peuvent sembler banales. Enlever les décimales peut résulter en une erreur de BPM, dans ce cas-là.
```